### PR TITLE
prevent progress tracker skipping percent played on short videos

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

- Fixes bug where progress tracker polls every 1s and sometimes misses percentages played on short videos (eg. would record 24% and 26% but miss 25% and therefore fail to track 25% played)

## What is the value of this and can you measure success?

- Accurate tracking

## Does this affect other platforms - Amp, Apps, etc?

No

## Screenshots

N/A

## Request for comment

@SiAdcock this fixes the bug we saw earler!

### CODE REVIEW

File: static/src/javascripts/bootstraps/enhanced/youtube.js [hunk 1]
```diff
@@ -59,18 +59,17 @@ define([
         var currentTime = player.getCurrentTime();
         var percentPlayed = Math.round(((currentTime / player.duration) * 100));
 
-        if (percentPlayed > 0 && percentPlayed < 100 &&
-            percentPlayed % 25 === 0,
-            players[atomId].trackingCalls.indexOf(percentPlayed) === -1) {
-            players[atomId].trackingCalls.push(percentPlayed);
-            tracking.track(percentPlayed, atomId);
+        if (players[atomId].pendingTrackingCalls.length &&
```
Comment (SiAdcock) [inline static/src/javascripts/bootstraps/enhanced/youtube.js]: Maybe we could extract `players[atomId].pendingTrackingCalls` as a variable?

Comment (GHaberis) [inline static/src/javascripts/bootstraps/enhanced/youtube.js]: Yep, good point, done!

Comment (GHaberis) [inline static/src/javascripts/bootstraps/enhanced/youtube.js]: Hi @SiAdcock! Is it OK to merge this now I've made this change?

**Comment by SiAdcock** [comment]
👍

**Pull request merged**